src/components/Dropdown/Dropdown.tsx

```diff
@@ -546,18 +562,17 @@ const Dropdown = ({
         {closeIcons}
       </Button>
       {isOpen && (
-        <StyledOptionsContainer
-          color={defaultedColor}
-          variant={optionsVariant}
-          role="listbox"
-          ref={mergeRefs([
-            optionsContainerRef,
-            optionsContainerInternalRef,
-            optionsScrollListenerCallbackRef,
-          ])}
-          {...optionsContainerProps}
-        >
-          {options.map(option => (
+        <Virtuoso
+          data={options}
+          rangeChanged={(range: ListRange) => setScrollIndex(range.startIndex)}
+          initialTopMostItemIndex={rememberScrollPosition ? scrollIndex : 0}
+          initialItemCount={
+            typeof window !== 'undefined' && window.document && window.document.createElement
+              ? initialOptionCount
```

Comment:
Why do we need this? Does options.length not work in SSR/test environments? Either way it probably needs a different name if it's optional. As-is I would be confused as to if I needed to supply it or not and in which situations I should/shouldn't.

Reply:
[Here](https://github.com/petyosi/react-virtuoso/issues/21)'s a thread on some potential SSR issues. I suppose this logic can be removed in favor of the user being able to supply a `virtualizeOptions={false}` prop if there are any complications with the virtualization.

Reply:
What would be a good value here? Just what we think will fit into the options container window initially?